include/GooseEYE/path.hpp

```diff
@@ -171,6 +175,69 @@ xt::xarray<int> path(const std::vector<int> &xa, const std::vector<int> &xb, pat
   }
 
   return xt::adapt(ret, {static_cast<size_t>(nnz), static_cast<size_t>(ndim)});
+  */
+}
+
+// -------------------------------------------------------------------------------------------------
+
+// https://www.geeksforgeeks.org/bresenhams-algorithm-for-3-d-line-drawing/
+void bressenham(
```

Comment:
I would place this in the `detail:` namespace. I don't think the end-user should interface with it.